### ISSUE TITLE
A helper named "step-manager" could not be found

### ISSUE THREAD

**Issue by herzzanu** [alexlafroscia/ember-steps#60]
I'm using this addon on 2 different projects. One project uses ember 2.7 and the other 2.9. On the 2.7 version it works fine but on the 2.9 version I get the error `Assertion Failed: A helper named "step-manager" could not be found`

Any idea why this is happening?

**Comment by alexlafroscia** [comment]
Sorry, no idea. Only thing I can think of would be that the add-on isn't installed correctly -- have you tried nuking `node_modules` and re-installing?

**Comment by herzzanu** [comment]
Thanks @alexlafroscia tried that and still doesn't work. Not sure what could be the issue.

**Comment by alexlafroscia** [comment]
Yeah, me neither. Any possibility you can upload the repo somewhere that I can take a look? Or maybe you could just post the bit of HBS that's invoking the step manager, to start?

**Comment by KamiKillertO** [comment]
Hello I'm facing the same error.
My project use ember 2.11 and the problem appear after I've move to the pods structure

**Comment by KamiKillertO** [comment]
I've manage to fix my problem by removing `podModulePrefix` in my `config.js`

**Comment by alexlafroscia** [comment]
Hmm, that's interesting... There's nothing inherently "different" about the components in this addon that woould make the `step-manager` component unavailable based on the pod config.  Can you post a repo reproducing the issue @KamiKillertO?

**Comment by KamiKillertO** [comment]
Done 👍 
https://github.com/KamiKillertO/ember-steps-issue-60

Step-manager works in `app/templates/application.hbs` but not in `app/pods/application/template.hbs`

**Comment by alexlafroscia** [comment]
Huh, seems really strange. I appreciate you making a reproduction for me, I'll dig into it and figure out what's up.

**Comment by alexlafroscia** [comment]
Hey @KamiKillertO, so I think the issue in that repo is with the way that the `environment.js` file is configured.  I found two different wants to "fix" that test repo:

- Remove the `podModulePrefix` configuration
- Move your `pods` directory to `step-manager-test/pods` instead

Based on [this](http://cball.me/organize-your-ember-app-with-pods/) blog post that I found when looking into it, it seems that the configuration option that you provided did not match the location that you actually placed your pods.  If you just leave that property empty, it will default to the location that you actually had them, so I would probably just do that.

**Comment by KamiKillertO** [comment]
Hello,

I've try both and it's still not working 😞.

I've create the template using ember-cli -> `ember g template application`
Output :
```
installing template
  create app/pods/application/template.hbs
```

My template looks like this :

```handlebars
<h1>Steps</h1>
{{#step-manager as |w|}} 
  {{#w.step name='a'}} 
    This is the first step! 
 
    <button {{action w.transition-to 'b'}}> 
      Next, please! 
    </button> 
  {{/w.step}} 
 
  {{#w.step name='b'}} 
    This is the second step! 
 
    <button {{action w.transition-to 'a'}}> 
      Wait, go back! 
    </button> 
  {{/w.step}} 
{{/step-manager}}
```
At this point step-manager is still not working.
I've try to create a component step-manager which only export you component:

```javascript
/* app/pods/components/step-manager/component.js */
export { default } from 'ember-steps/components/step-manager/component';
```
This make me move forward and now I've another issue 
```
ember.debug.js:17636 Assertion Failed: The component helper cannot be used without a valid component name. You used "step-manager/step" via (component "step-manager/step")
```

**Comment by alexlafroscia** [comment]
Man, this is really frustrating. The only thing that I can think of is that the Pod settings of your app and my addon are in conflict. The resolver is set up to look for pods under the `pod` prefix in your application (I can see that the template was generated under `app/pods/application/template.hbs`) but the addon is hosting the pod within the root of the `app` directory.

**Comment by willrax** [comment]
I've hit this same issue. I solved it by doing this in my own application.

`app/pods/components/step-manager/step/component.js`
```js
import StateManagerComponent from 'ember-steps/components/step-manager/component';

export default StateManagerComponent;
```

`app/pods/components/step-manager/component.js`
```js
import StepComponent from 'ember-steps/components/step-manager/step/component';

export default StepComponent;
```

**Comment by alexlafroscia** [comment]
I'll add to my todo-list to just refactor this into a "regular" component instead. I keep forgetting but will add it to my todo list. Unfortunately, it's kind of hard for Ember to test an addon in multiple different application scenarios.

**Comment by willrax** [comment]
I can put together a PR for you if you like.

**Comment by alexlafroscia** [comment]
Sure! That would be great. Plus, you can confirm that it fixes your use case.

**Comment by willrax** [comment]
Sounds good. I'll pull it together tonight

**Comment by alexlafroscia** [comment]
This should be closed by #72, with which I will release a new version right now.
